src/components/FilteredOrgList.js

```diff
@@ -14,7 +14,7 @@ const FilteredOrgList = (props) => {
     return (
       <div
         index={index}
-        key={org.id}
+        key={`${org.type}.${org.id}`}
```

Comment:
need this to key the keys unique.  district id's overlap with priority id's and priority type values